src/BenchmarkDb/Program.cs

```diff
@@ -35,7 +35,7 @@ public static class Program
             = new Dictionary<string, DriverBase>
             {
                 { "ado-npgsql", new AdoDriver(NpgsqlFactory.Instance) },
-                { "ado-mysql", new AdoDriver(MySqlClientFactory.Instance) },
+                { "ado-mysql", new NpgsqlAdoDriver() },

```

Comment:
Should this be changed in the prev line (i.e. for `ado-npgsql` instead of `ado-mysql`)?

Reply:
Oops... that was a tad too quick...

Reply:
Haha, the one time...